Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5811,6 +5811,22 @@ PitchCorrectionAlgorithm:
     WebCore:
       default: MediaPlayerEnums::PitchCorrectionAlgorithm::BestAllAround
 
+PointerLockEnabled:
+  type: bool
+  status: internal
+  category: dom
+  condition: ENABLE(POINTER_LOCK)
+  humanReadableName: "Pointer Lock API"
+  humanReadableDescription: "Enable the Pointer Lock API (Locking/Unlocking/Events)"
+  defaultValue:
+    WebKitLegacy:
+      default: false
+    WebKit:
+      "PLATFORM(MAC)": true
+      default: false
```

Comment:
Is this right for other ports, where `ENABLE(POINTER_LOCK)` is currently 1?

Reply:
Good catch. Clearly not

Reply:
I'll do this instead:

```
"PLATFORM(IOS_FAMILY)": false
default: true
```